about.html

```diff
@@ -46,7 +46,7 @@
       Happy Coding!
       <br>
       <br>
-      <pre> while(!(succeed = try () ));</pre>
+      <pre> while(!succeed){ try();}</pre>
```

Comment:
Oh!! This code is correct. What this does is it calls the function `try()`, stores its value in `succeed` and then checks the condition `!success`. This keeps repeating till `succeed` is `true`.